config.yml

```diff
@@ -134,6 +134,19 @@
 #
 ##  Retry interval between failed bulk attempts
 #elasticsearch.bulk.retry_interval: 10
+#
+#
+##  Minimal interval of time between MemQueue checks for being full
+#elasticsearch.bulk.mem_queue.refresh_interval: 1
```

Comment:
Why is `mem_queue` under `elasticsearch.bulk`? I'd have expected it under `service`. I thought that everything in `elasticsearch.bulk` is being passed directly to the ES client?

Reply:
Ah ok, it's because we already have mem_queue stuff here?
Looks like the other configs are: `queue_max_mem_size` and `queue_max_size`. So should these be `queue_refresh_interval` and `queue_refresh_timeout` for consistency?

Reply:
Good call! Changed in https://github.com/elastic/connectors/pull/2079/commits/eacbb234894692f49070c96ef5cdd466018be093